config/configgrpc/configgrpc.go

```diff
@@ -150,6 +180,13 @@ type ServerConfig struct {
 	IncludeMetadata bool `mapstructure:"include_metadata"`
 }
 
+// NewDefaultServerConfig() creates a new ServerConfig with any default values set
+func NewDefaultServerConfig() *ServerConfig {
+	return &ServerConfig{
```

Comment:
This function will need to call `confignet.NewDefaultAddrConfig`

Reply:
Understood, will add it immediately